### DISCUSSION TITLE
Presentation - How GCAM works: - Markets, logits, share weights, and calibration

### DISCUSSION THREAD

**Discussion by DLoughlin** [DLoughlin/GLIMPSE-CE#111] [Show and tell]
On Aug. 28th, 2026, Dan Loughlin presented "How GCAM works - Markets, logits, share weights, and calibration" as part of the GLIMPSE/GCAM-USA State Applications Community of Practice webinar series. 

Slides: [HowGCAMWorks-2026.08.30.pptx](https://github.com/user-attachments/files/31619056/HowGCAMWorks-2026.08.30.pptx)
Recording: [Link](https://drive.google.com/file/d/1wSp7YeA2Z1b74AuyD8DuKpGsbSdEv7zA/view?usp=sharing)

Description: GCAM's simulation process involves stepping through time, and in each step finding the set of prices at which supply equals demands across thousands of markets simultaneously. In doing so, GCAM simulates the market shares for technologies using a logit equation. Important parameters in this equation include the levelized cost of each technology competing within the market, technology-specific share weights that represent bias and unmodeled issues, and a market-specific logit exponent that captures sensitivity to cost differences. Policies also typically operate through the logit function by impacting technology costs through taxes or subsidies. Understanding how the logit equation works, as well as its strengths and limitations, are critical skills for GCAM modelers. In this presentation, Dan Loughlin will describe market operation in GCAM and provide an example related to passenger car electrification.
